### DISCUSSION TITLE
Rendering issues on MacOS terminal

### DISCUSSION THREAD

**Discussion by bgourlie** [helix-editor/helix#605] [Q&A] [answered]
I've noticed rendering issues on the MacOS terminal. I'm on a M1 MacBook Air with the latest version of MacOS, and terminal theme is the default "Basic" theme.

I'm hesitant to file an issue for this because I can't entirely rule out an issue on my end—it's possible I've messed with some settings but can't recall what they are. Has anyone else encountered rendering issues on the MacOS terminal? I should note that rendering works as-expected in the IntelliJ IDEA terminal emulator.

<img width="1001" alt="Screen Shot 2021-08-17 at 10 20 19 AM" src="https://user-images.githubusercontent.com/996556/129753996-7147b2cb-79a7-45e1-b9cf-376e4608c16b.png">

**Comment by archseer** [comment, marked as answer]
I think you're lacking true color support: https://gist.github.com/XVilka/8346728

It worked for me in iTerm 2 but I guess the themes should have some fallback too. The MacOS terminal seems to be one of the few remaining ones that doesn't have this built-in (even Powershell has it)

**Reply by bgourlie** [reply]
That is definitely the issue. I keep trying to mark your response as the answer but it’s not doing anything.

**Reply by igilham** [reply]
I just found this issue, too. It might be worth documented the true colour requirement early in the README to save people time figuring this out again.

**Reply by sudormrfbin** [reply]
It was added to the [troubleshooting wiki](https://github.com/helix-editor/helix/wiki/Troubleshooting#rendering-issues-on-macos-terminal), though direct links to the FAQ and troubleshooting page would be nice.
